climada/entity/exposures/test/test_base.py

```diff
@@ -264,6 +265,35 @@ def test_io_hdf5_pass(self):
             self.assertEqual(point_df.x, point_read.x)
             self.assertEqual(point_df.y, point_read.y)
 
+    def test_write_raster_pass(self):
+        """Test write_raster function."""
+        test_file = Path(DATA_DIR, "test_write_raster.tif")
+        exp = Exposures.from_raster(HAZ_DEMO_FL, window=Window(10, 20, 50, 60))
+        exp.write_raster(test_file, 'value')
+        read_meta, read_data = u_coord.read_raster(test_file)
+        self.assertEqual(read_meta['transform'], exp.meta['transform'])
+        self.assertEqual(read_meta['width'], exp.meta['width'])
+        self.assertEqual(read_meta['height'], exp.meta['height'])
+        self.assertTrue(u_coord.equal_crs(read_meta['crs'], exp.meta['crs']))
+        self.assertEqual(read_data.shape, (1, exp.gdf.shape[0]))
+        np.testing.assert_array_equal(read_data[0], exp.gdf['value'])
```

Comment:
```suggestion
        self.assertEqual(read_data.shape, (1, exp.gdf.shape[0]))
        np.testing.assert_array_equal(read_data[0], exp.gdf['value'])
        test_file.unlink()
```
Delete the file after the test.